x-pack/plugins/beats_management/public/components/table/table.tsx

```diff
@@ -16,12 +16,21 @@ import { ControlBar } from './controls';
 import { TableType } from './table_type_configs';
 
 interface TableProps {
-  assignmentOptions: any[] | null;
-  assignmentTitle: string | null;
+  assignmentOptions?: any[] | null;
+  assignmentTitle?: string | null;
   items: any[];
   renderAssignmentOptions?: (item: any, key: string) => any;
   showAssignmentOptions: boolean;
   type: TableType;
+
+  isLoadingSuggestions: boolean;
```

Comment:
Similar comment as in the `controls` file about passing these props through. Could we define an interface for these and pass them through as an object, or include the interface definition when assigning the props type to the class, to keep the table's props definitions cleaner, and only related to what it needs to function?

Reply:
similar answer 😊